public/src/index.jsx

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+import ReactDom from 'react-dom';
+import App from './component/App';
```

Comment:
component should be components, unless you're only planning on putting one component in this folder